packages/context/test/acceptance/tagged-bindings.feature.md

```diff
@@ -20,7 +20,48 @@ let ctx = new Context();
 let binding = ctx
   .bind('foo')
   .to('bar')
-  .tag('qux');
+  .tag('controller');
 
-console.log(binding.tags); // => Set { 'qux' }
+console.log(binding.tagNames); // =>  ['controller']
+```
+
+## Scenario: Multiple named tags
+
+- Given a context
+- And a binding named `foo` with value `bar`
+- When I tag it `controller` and `rest`
+- Then it should be tagged with `controller` and `rest`
+
+```ts
+// create a container for bindings
+let ctx = new Context();
+
+// create a tagged binding
+let binding = ctx
+  .bind('foo')
+  .to('bar')
+  .tag('controller', 'rest');
+
+console.log(binding.tagNames); // =>  ['controller', 'rest']
+```
+
+## Scenario: Tags with both name and value
+
+- Given a context
+- And a binding named `foo` with value `bar`
+- When I tag it `{name: 'my-controller'}` and `controller`
+- Then it should be tagged with `{name: 'my-controller', controller: 'controller'}`
+
+```ts
+// create a container for bindings
+let ctx = new Context();
+
+// create a tagged binding
+let binding = ctx
+  .bind('foo')
+  .to('bar')
+  .tag({name: 'my-controller'});
```

Comment:
I think this should be `.tag({name: 'my-controller'}, 'controller')`

Reply:
Fixed